docs/backend/api/README.md

```diff
@@ -0,0 +1,83 @@
+# API Reference
+
+## Devices
+
+### Configure Devices
+
+    POST /api/devices
+
+Creates the device configuration file.
+
+### Parameters
+#### Header Parameters
+
+
+Field | Required | Description
+--- | --- | ---
+Authorization | Y | Session token.
+
+#### Body Parameters
+
+Field | Required | Description
+--- | --- | ---
+ThingD | Y | The ThingD configuration data that's sent by the user.
```

Comment:
hahaha. It would be quite boring describe all the fields here, right? 😛 
Please, add a reference to the [`knot-virtualthing`](https://github.com/CESARBR/knot-virtualthing/blob/master/confs/device.conf) device.conf example.

Reply:
Done!